srcpkgs/cross-powerpc-linux-gnu-libc/template

```diff
@@ -0,0 +1,351 @@
+# Template build file for 'cross-powerpc-linux-gnu'
+#
+_binutils_version=2.29.1
+_gcc_version=7.3.0
+_glibc_version=2.26
+_linux_version=4.9.8
+
+_triplet=powerpc-linux-gnu
+_archflags="-mcpu=powerpc"
+_sysroot="/usr/${_triplet}"
+
+pkgname=cross-${_triplet}
+version=0.25
+revision=1
+short_desc="GNU Cross toolchain for the ${_triplet} target (binutils/gcc/glibc)"
+maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
```

Comment:
should be under your maintainer name and email

Reply:
Oops, sorry these were meant for a different fork, not the main repo.